Microsoft.Toolkit.Uwp.UI.Behaviors/VoiceCommands/WindowsMediaSpeechRecognizer.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
```

Comment:
You're missing the headers on the files here, so CI is still failing. You can run the `UpdateHeaders` command in the `build` directory to help, though if it somehow touches more files beyond yours, don't commit those.

The build script in the build directory can also do a local test build as if it were in the CI, so that can help to validate.

Reply:
Headers updated. I forgot that I had to do this. I'm a bit rusty, sorry.